.gitignore

```diff
@@ -11,3 +11,6 @@ out/
 # gradle
 .gradle
 build/
+
```

Comment:
I'm fine with removing them.  It's just nice to not have to constantly manually ignore .swp files.  Does no one else use vim to edit files in this repo?